scopes/preview/ui/component-preview/preview.tsx

```diff
@@ -35,14 +35,51 @@ export interface ComponentPreviewProps extends Omit<IframeHTMLAttributes<HTMLIFr
 // TODO - Kutner fix unused var - 'hotReload' should be used
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 export function ComponentPreview({ component, previewName, queryParams, hotReload, ...rest }: ComponentPreviewProps) {
```

Comment:
[critical]
check this! it will probably effect preview cards too!
maybe add a flag - `fullContentHeight`